models/task/task.go

```diff
@@ -672,6 +672,10 @@ func (t *Task) ExportTaskInfos(values *values.Values) {
 	if t.ResolverUsername != nil {
 		m["resolver_username"] = t.ResolverUsername
 	}
+	m["watcher_usernames"] = t.WatcherUsernames
+	if t.WatcherGroups != nil && len(t.WatcherGroups) > 0 {
```

Comment:
First check `t.WatcherGroups != nil` can be omitted, since the second works on `nil` slices.

Reply:
done